website/docs/docs/collaborate/govern/model-access.md

```diff
@@ -163,16 +163,38 @@ Of course, dbt can facilitate this by means of [the `grants` config](/reference/
 
 As we continue to develop multi-project collaboration, `access: public` will mean that other teams are allowed to start taking a dependency on that model. This assumes that they've requested, and you've granted them access, to select from the underlying dataset.
 
-### What about referencing models from a package?
+### How do I ref a model from another project?
 
-For historical reasons, it is possible to `ref` a protected model from another project, _if that protected model is installed as a package_. This is useful for packages containing models for a common data source; you can install the package as source code, and run the models as if they were your own.
+<VersionBlock lastVersion="1.5">
+
+In dbt Core v1.5, the only way to reference a model from another project is by installing that project as a package. There is no way to enforce access distinctions between `protected` and `public` models.
+
+This changes in v1.6. Select it from the dropdown above.
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.6">
+
+It is possible to `ref` a model from another project in two ways:
+1. As a "project" dependency, via "cross-project `ref`" (a feature of dbt Cloud Enterprise)
+2. As a "package" dependency, whereby all source code from that project is installed into your own
```

Comment:
Looks like we just need to finish this thought - own project?

Reply:
I definitely meant to write `your own project` here — although we're using `project` to mean so many different things. It's almost more like:
- your own working directory
- your own environment
- your own dbt runtime
- ...

Reply:
I like `environment`!